ProcessLib/Process.h

```diff
@@ -162,7 +232,10 @@ class Process
 	}
 
 protected:
+	unsigned const _integration_order = 2;
```

Comment:
The variable is declared using const qualifier. How is it possible to changed the value later on?